llarp/handlers/tun.cpp

```diff
@@ -343,12 +343,6 @@ namespace llarp
         }
         else if(addr.FromString(qname, ".snode"))
         {
-          if(HasPathToSNode(addr.as_array()))
-          {
-            msg.AddINReply(ObtainIPForAddr(addr, true));
-            reply(msg);
-            return true;
-          }
```

Comment:
why did this get taken out?

Reply:
the reply is deferred to when we have the ability to send to the service node now.